src/Api/Data/StoryblokLink.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Storyblok\Api\Data;
+
+final class StoryblokLink
+{
+	private int $id;
+	private string $uuid;
+	private string $slug;
+
+	/**
+	 */
+	public function __construct (array $data)
+	{
+		$this->id = $data["id"];
+		$this->uuid = $data["uuid"];
+		$this->slug = $data["slug"];
+	}
+
+	/**
+	 */
+	public function getId () : int
+	{
+		return $this->id;
+	}
+
+	/**
+	 */
+	public function getUuid () : string
+	{
+		return $this->uuid;
+	}
+
+	/**
+	 */
+	public function getSlug () : string
+	{
+		return $this->slug;
+	}
+}
```

Comment:
I'd probably mark this class as `readonly` and make the fields public, so that we get rid of the getters.